src/Features/Core/Portable/InitializeParameter/AbstractAddParameterCheckCodeRefactoringProvider.cs

```diff
@@ -790,40 +953,45 @@ private SyntaxNode CreateArgumentException(
     {
         var text = methodName switch
         {
-            nameof(string.IsNullOrEmpty) => new LocalizableResourceString(nameof(FeaturesResources._0_cannot_be_null_or_empty), FeaturesResources.ResourceManager, typeof(FeaturesResources)).ToString(),
```

Comment:
I don't understand the move with creating a localizable string and then immideatelly invoking `ToString` on it. This should be the same as just accessing the resource directly. Am I correct?